<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>358</width>
    <height>452</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>To Do List</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QSlider" name="slider_opacity">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>390</y>
      <width>161</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="pageStep">
     <number>1</number>
    </property>
    <property name="value">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_opacity">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>390</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>100</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Reset_Opacity">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>390</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <widget class="QCalendarWidget" name="Calender">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>331</width>
      <height>201</height>
     </rect>
    </property>
   </widget>
   <widget class="QDateEdit" name="date_move">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="button_calende_move">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>0</y>
      <width>30</width>
      <height>31</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>해당 날짜로 이동</string>
    </property>
    <property name="text">
     <string>이동</string>
    </property>
   </widget>
   <widget class="QListView" name="list_print">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>331</width>
      <height>131</height>
     </rect>
    </property>
   </widget>
   <widget class="QToolButton" name="button_option">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>0</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>설정</string>
    </property>
   </widget>
   <widget class="QToolButton" name="butto_grapy">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>0</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>얼마나 실천했을까?</string>
    </property>
    <property name="text">
     <string>통계</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_add_plan">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>0</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>해당 날짜에 일정추가</string>
    </property>
    <property name="text">
     <string>일정 추가</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>358</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuTO_DO_LIST">
    <property name="title">
     <string>TO DO LIST</string>
    </property>
   </widget>
   <addaction name="menuTO_DO_LIST"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
